src/components/spatial/Spatial.js

```diff
@@ -196,17 +186,22 @@ export default class Spatial extends AbstractSelectableComponent {
   createTileLayer(layer) {
     const [layerType, source] = layer;
```

Comment:
What is `layerType` here? Is it the channel or more akin to how decl.gl defines a layer? Maybe consider a new variable name to clarify this.

Reply:
i think it's just the name of the layer (like 'polyT')